### DISCUSSION TITLE
New Watch Support ?

### DISCUSSION THREAD

**Discussion by Odroid13** [NightscoutFoundation/xDrip#3443] [Ideas]
support for new connected watch?  Hello, I am a user of a Huawei FIT connected watch because the screen is larger than Mi Band and it remains comfortable for my eyes.  I wanted to know if it would be possible to add support for this watch?  If this is not possible can you tell me if all Mi Band versions are compatible with Xdrip+ including the latest version?  Thank you so much

**Comment by Navid200** [comment]
Please contact WatchDrip for MiBand support.
The watch may already be supported or they can tell you when.

**Reply by PhilTU** [reply]
Miband 8 from Xiaomi I think is not compatible. 7 works with a hack to show it as a ZeppOS device.

Huawei is a different sport.
